app/controllers/users_controller.rb

```diff
@@ -68,11 +68,9 @@ def update
   def edit
     @action = "update" # sets the form url
     if params[:id] # admin only
```

Comment:
Could anyone please tell me what the comment means here?

Reply:
Hi @seafr .
I guess if you will see comment in line number 77 , then it seems this route was made to give **admins** the power to edit profile of **any** user .
This route : 
<img width="532" alt="screen shot 2018-03-28 at 11 49 56 pm" src="https://user-images.githubusercontent.com/14952645/38048842-331528fc-32e4-11e8-9c55-6d4722dc174b.png">

Reply:
I think currently admins do not require the ability to edit the profile of any user , hence it is commented out .

Reply:
Yes that's what I was thinking. So I think this comment shoud be removed now as it is misleading.

Reply:
Ok...so if you are not signed in and you go to **/profile/edit** page , we get page not found error . Do you think we should be redirected to the **login-screen** or may be show a flash ?
Would you like to open a first-timer issue ?

Reply:
According to the routes file, **/profile/edit** routes to `users#edit`, which is the same *edit* action above. Let me test this and get back to you.

Reply:
Yes, I think the user should be redirected to **/login**. The flash should also be displayed. 
It's a good idea to open an FTO issue for that as it involves a tiny modification. I'll try it.

Reply:
Awesome !

Reply:
I did it (not without glitches though :( ). Thank you for the suggestion.